pkg/cephfs/volume.go

```diff
@@ -63,6 +78,19 @@ func createVolume(volOptions *volumeOptions, adminCr *credentials, volID volumeI
 
 	if pathExists(volRoot) {
 		klog.V(4).Infof("cephfs: volume %s already exists, skipping creation", volID)
+		size, err := getVolumeAttribute(volRoot, "ceph.quota.max_bytes")
```

Comment:
If path does not exist, the call from ControllerExpandVolume would go ahead and attempt to create the path and perform other operations as needed. It is better to deal with an expand request as a separate call, and not fold the operation into createVolume, keeping abstractions clean. The code reuse (by using createVolume) seems minimal and hence not really needed.